app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,24 @@
  *= require_tree .
  *= require_self
  */
+
+img {
+   width: 300px
+ }
+
+h1 {
+  color: blue
+}
+
+ul {
+  list-style-type: none;
+}
+
+input[type="submit"] {
+    background: none;
+    border: none;
+    outline: none;
+    text-decoration: underline;
+    color: blue;
+    cursor: pointer;
+  }
```

Comment:
Moved my stying to the CSS file. Moving forward I hope to style by so my views can be more unique.